feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailViewModel.kt

```diff
@@ -93,4 +175,16 @@ class MovieDetailViewModel @Inject constructor(
       }
     }
   }
+
+  fun getRateColor() = when (_state.value.videoDetail?.voteAverage ?: 0.0F) {
+    in 0.0..0.33 -> {
+      Color.Red
+    }
+
+    in 0.34..0.66 -> {
+      Color.Yellow
+    }
+
+    else -> Color.Green
+  }
```

Comment:
If we agree that view models are responsible for holding UI business logic, consider relocating this function to the composable (UI) files or even to the `core:ui module`. Additionally, it could be converted into an extension function for Float.

```suggestion
fun Float?.toRatingColor() = when (this) {
    null -> Color.Gray
     in 0.0..0.33 -> Color.Red
     in 0.34..0.66 -> Color.Yellow
     else -> Color.Green
}
```